scripts/build_release_upload.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+. ${HOME}/build_env
+set -e
+set -x
+
+# persistent storage of repo manager scratch space is on EFS
+if [ ! -z "${AWS_EFS_MOUNT}" ]; then
+    if mount|grep -q /data; then
+	echo /data already mounted
+    else
+	sudo mkdir -p /data
+	sudo mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport "${AWS_EFS_MOUNT}":/ /data
+	# make environment for release_deb.sh
+	sudo mkdir -p /data/_aptly
+	sudo chown -R ${USER} /data/_aptly
+	export APTLY_DIR=/data/_aptly
+    fi
+fi
+
+cd ${GOPATH}/src/github.com/algorand/go-algorand
+
+git push origin --tags ${BRANCH}
```

Comment:
It seems recommend to just push a specific tag instead of all tags - can we do that here?

Reply:
updated to push current branch and specific tag